app/views/journeys/_published.html.erb

```diff
@@ -2,7 +2,9 @@
   <div class="govuk-grid-column-two-thirds">
     <div class="govuk-body sdn-content">
       <h2 class="govuk-heading-l">Úvodné informácie</h2>
-      <p class="govuk-body-s">Aktualizované: <%= @journey.updated_at.strftime('%d.%m.%Y') =%></p>
+      <% if @journey.last_check  %>
```

Comment:
Mas screen ako to vyzera? Podla mna to nechceme davat nejako takto extra hore, aby to rusilo.

Reply:
Dohodli sme, ze zatial zostava tak ako je